src/main/java/nextstep/optional/Expression.java

```diff
@@ -14,12 +16,14 @@ private static boolean matchExpression(Expression e, String expression) {
     }
 
     static Expression of(String expression) {
-        for (Expression v : values()) {
-            if (matchExpression(v, expression)) {
-                return v;
-            }
-        }
 
-        throw new IllegalArgumentException(String.format("%s는 사칙연산에 해당하지 않는 표현식입니다.", expression));
+        return Arrays.stream(Expression.values())
+                .filter(v -> matchExpression(v, expression))
+                .findFirst()
+                .orElseThrow(() -> {
+                    throw new IllegalArgumentException(String.format("%s는 사칙연산에 해당하지 않는 표현식입니다.", expression));
+                });
```

Comment:
```suggestion
                .orElseThrow(() -> new IllegalArgumentException(String.format("%s는 사칙연산에 해당하지 않는 표현식입니다.", expression)));
```
단일행이니 굳이 문(statement)으로 표현하기보다는 식(expression)으로 표현하는 것은 어떨까요